suite-native/atoms/src/TransactionItem.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
```

Comment:
This file is unused and deprecated, you can delete it

Reply:
deleted in dc4a16df4